zebra-rpc/src/methods.rs

```diff
@@ -270,3 +313,10 @@ pub struct GetBlock {
     #[serde(with = "hex")]
     data: SerializedBlock,
 }
+
+#[derive(serde::Serialize)]
+/// Response to a `getblock` RPC request.
+pub struct GetBestBlockHash {
+    #[serde(with = "hex")]
+    hex: SerializedBlockHash,
```

Comment:
The hash data not actually hex-encoded until we serialize it.

Would you mind changing the name so it's clearer?

```suggestion
    hash: SerializedBlockHash,
```

Reply:
I am not so sure about this because `hex` is the response the call is waiting for:

https://zcash.github.io/rpc/getbestblockhash.html

Lightwalletd may work with any field name (i didn't checked) but i think we should remain compatible with the zcashd rpc methods the most we can.

Reply:
I don't mind what you call this field, feel free to resolve this conversation.

But I wanted to explain what lightwalletd is parsing here.

Lightwalletd expects a JSON string, not a JSON field:
```go
		var hashHex string
		err = json.Unmarshal(result, &hashHex)
```

https://github.com/zcash/lightwalletd/blob/master/common/common.go#L304-L311

Strings look like:
> Result
> "hex"      (string) the block hash hex encoded 

https://zcash.github.io/rpc/getbestblockhash.html

And fields look like:

> Result:
> {
>   "chain": "xxxx",        (string) current network name as defined in BIP70 (main, test, regtest)
>   "blocks": xxxxxx,         (numeric) the current number of blocks processed in the server
>   ...
> }

https://zcash.github.io/rpc/getblockchaininfo.html

```go
	var getblockchaininfoReply ZcashdRpcReplyGetblockchaininfo
	err := json.Unmarshal(result, &getblockchaininfoReply)
```

https://github.com/zcash/lightwalletd/blob/master/common/mempool.go#L146-L151

Reply:
Sorry, my mistake, i was not reading correctly the rfc responses. I got this fixed now.